SPRCLI/Sources/SPR/Process+SPR.swift

```diff
@@ -6,7 +6,6 @@
 //
 
 import Foundation
-import PackageDescription
```

Comment:
This is an old remnant of the AWSSDKSwiftCLI dependency on the `swift-package-manager` package.